hosts.example

```diff
@@ -29,5 +29,14 @@ graphite_fqdn=tendrl.example.com
 #ansible_become=yes
 #ansible_user=cloud-user
 
+# Uncomment to configure etcd to use client to server authentication with HTTPS 
+# client certificates, as well as all Tendrl components will be reconfigured 
+# accordingly.  Default set to false.
```

Comment:
For this comment to be fully clear, we will need to add much more information there.

Reply:
@mbukatov Both of these variables are already mentioned in the README.

The readme pretty much says "etcd tls client authentication" for the etcd_tls_client_auth var.